sensor/kubernetes/eventpipeline/component/component.go

```diff
@@ -0,0 +1,16 @@
+package component
+
+import "github.com/stackrox/rox/generated/internalapi/central"
+
+// PipelineComponent components that constitute the eventPipeline
+type PipelineComponent interface {
+	Start() error
+	Stop(error)
+	ResponsesC() <-chan *central.MsgFromSensor
+}
+
+// OutputQueue component that redirects Resource Events and Alerts to the output channel
+type OutputQueue interface {
```

Comment:
Add
```
// go:generate mockgen-wrapper
```
And run the code generation step (I think `make go-generated-srcs`?) to generate the gomock implementation of this other than implementing it by hand.

Reply:
done